invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/components/RecordsResultsListItem.js

```diff
@@ -82,7 +82,7 @@ class RecordsResultsListItem extends Component {
         numOtherVersions={numOtherVersions}
       >
         <Item key={key ?? result.id}>
-          <Item.Content>
+          <Item.Content className="max-width-100">
```

Comment:
I would add it to item.overrides directly instead

as in
```
.item > .content{
  max-width: 100%
}

it will fix it for any other component which overrides the ResultListItem. 
In case of how you implemented it it will work only for these 3 cases.

Reply:
Ok, makes sense, it will affect all the Item cmp, but I guess it's fine